apps/services/sessions/src/app/sessions/sessions.controller.spec.ts

```diff
@@ -45,11 +57,219 @@ describe('SessionsController', () => {
       app.cleanUp()
     })
 
+    it('GET /v1/me/sessions should return first page of user sessions in descending order', async () => {
+      // Act
+      const res = await server.get(`/v1/me/sessions`)
+
+      // Assert
+      expect(res.status).toEqual(200)
+      expect(
+        res.body.data.every(
+          (session: Session) =>
+            session.actorNationalId === user.nationalId ||
+            session.subjectNationalId == user.nationalId,
+        ),
+      )
+      expect(res.body.data).toMatchObject(
+        sortBy(sessions, 'timestamp')
+          .reverse()
+          .filter(
+            (session) =>
+              session.actorNationalId === user.nationalId ||
+              session.subjectNationalId == user.nationalId,
+          )
+          .map((session) => ({
+            id: session.id,
+            actorNationalId: session.actorNationalId,
+            subjectNationalId: session.subjectNationalId,
+            clientId: session.clientId,
+            timestamp: session.timestamp.toISOString(),
+            userAgent: session.userAgent,
+            ip: session.ip,
+          }))
+          .slice(0, 10),
```

Comment:
We should create some utility to do the sorting, filtering and slicing as this is repeated through out the tests.
We could also do some other utility for the expect above.

Also the `map` can be simplified if you only need to change the `timestamp`
```
.map((session) => ({
  ...session,
  timestamp: session.timestamp.toISOString()
}))
```

Reply:
The map is needed since we are comparing properties of the sequelize model to the response.

Don´t think repetition is necessarily bad in test code.  It can make it easier to understand an individual test at a glance without having to navigate utility functions.

Reply:
Ahh ofc the sequelize props.
Okay leave this as is for now.